src/setupProxy.js

```diff
@@ -0,0 +1,8 @@
+const { createProxyMiddleware } = require('http-proxy-middleware');
+module.exports = function(app) {
+  app.use(
+    createProxyMiddleware('http://localhost:5001/api/study-server', {
```

Comment:
@geofjamg I haven't changed it because I'm not sure there is **api/merge-server**

Reply:
it is merge-orchestrator-server